assets/js/components/notifications/CTA.js

```diff
@@ -45,6 +46,11 @@ const CTA = ( {
 			'googlesitekit-cta--error': error,
 		} ) }
 	>
+		{ headerText && (
+			<span className="googlesitekit-cta__header_text">
```

Comment:
Semantically, I would suggest this should be an `h2` rather than a `span`, considering the use of `h3` for the title below...